docs/using_pyodide_from_javascript.md

```diff
@@ -102,6 +101,101 @@ pyodide.loadPackage('matplotlib').then(() => {
 });
 ```
 
+## Alternative way to load packages and run Python code
+
+Alternatively you can run Python code without manually pre-loading packages. You can do this with [pyodide.runPythonAsync](api_reference.md#pyodideloadpackagenames-messagecallback-errorcallback) function, which will automatically download all packages that the code snippet imports.
+
+Note: although the function is called Async, it still blocks the main thread. To run Python code asynchronously see [WebWorker](using_pyodide_from_webworker.md)
+
+## Alternative Example
+
+```html
+<!DOCTYPE html>
+<head>
+    <script type="text/javascript">
+        window.languagePluginUrl = 'https://pyodide-cdn2.iodide.io/v0.15.0/full/';
+    </script>
+    <script src="https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js"></script>
+</head>
+
+<body>
+  <p>You can execute any Python code. Just enter something in the box below and click the button.</p>
+  <input id='code' value='sum([1,2,3,4,5])'>
+  <button onclick='evaluatePython()'>Run</button>
+  <br>
+  <br>
+  <div>
+    Output:
+  </div>
+  <textarea id='output' style='width: 100%;' rows='6' disabled></textarea>
+
+  <script>
+    const output = document.getElementById("output")
+    const code = document.getElementById("code")
+
+    function addToOutput(s) {
+      output.value+= `>>>${code.value}\n${s}\n`
+    }
+
+    output.value = 'Initializing...\n'
+    // init pyodide
+    languagePluginLoader.then(() => { output.value+='Ready!\n' })
+
+    function evaluatePython() {
+      pyodide.runPythonAsync(code.value)
+        .then(output => addToOutput(output))
+        .catch((err) => { addToOutput(err) })
+    }
+  </script>
+</body>
+
+</html>
+```
+
+## Accessing Python scope from JavaScript
```

Comment:
This part is a bit redundant with the https://pyodide.readthedocs.io/en/latest/type_conversions.html section but I suppose it's fine to have a longer explanation here as well. 

Maybe we should move some of these examples to a "Getting starting" page.